docs/en/sdk-manual/android.md

```diff
@@ -167,16 +167,16 @@ In order to improve the efficiency of querying and analysis, we apply limits to
 
 Automatically collected events:
 
-| Event name       | Triggered                                                                             | Event Attributes                                                                                                  |
-| ---------------- | ------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
-| _session_start   | when users app come to foreground for the first time and there is no on-going session | _session_id <br>_session_start_timestamp<br>_session_duration                                                     |
-| _screen_view     | when the activity callback `onResume()` method                                        | _screen_name<br>_screen_id<br>_previous_screen_name<br>_previous_screen_id<br>_entrances<br>_engagement_time_msec |
-| _app_exception   | when the app crashes                                                                  | _exception_message<br>_exception_stack                                                                            |
-| _app_update      | when the app is updated to a new version and launched again                           | _previous_app_version                                                                                             |
-| _first_open      | when the user launches an app the first time after installation                       |                                                                                                                   |
-| _os_update       | when device operating system is updated to a new version                              | _previous_os_version                                                                                              |
-| _user_engagement | when the app is in the foreground for at least one second                             | _engagement_time_msec<br>                                                                                         |
-| _profile_set     | when the `addUserAttributes()` or `setUserId()` API is called                         |                                                                                                                   |
+| Event name       | Triggered                                                                              | Event Attributes                                                                                                  |
+|------------------|----------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| _session_start   | when users app come to foreground for the first time and there is no on-going session  | _session_id <br>_session_start_timestamp<br>_session_duration                                                     |
```

Comment:
when user's app comes to the foreground for the first time and there is no ongoing session

Reply:
fixed.